src/NServiceBus.Core/Transports/Msmq/CheckEndpointNameComplianceForMsmq.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NServiceBus.Transports.Msmq
+{
+    using System;
+
+    class CheckEndpointNameComplianceForMsmq
```

Comment:
This could be a static class and method since there's no state it needs to maintain.